libp2p/stream/chronosstream.nim

```diff
@@ -72,26 +75,25 @@ proc trackPeerIdentity(s: ChronosStream) =
   if not s.tracked:
     if not isNil(s.peerInfo) and s.peerInfo.agentVersion.len > 0:
       # / seems a weak "standard" so for now it's reliable
-      let name = s.peerInfo.agentVersion.split("/")[0]
-      libp2p_peers_identity.inc(labelValues = [name])
+      s.shortAgent = s.peerInfo.agentVersion.split("/")[0]
```

Comment:
this might crash if it returns nil

Reply:
well should not happen as len is checked tho

Reply:
`"no slashes".split("/")[0] == "no slashes"` afaik, will test

Reply:
works

Reply:
I added a `strdefine` for `KnownLibP2PAgents`, can be now defined in user apps.